utils/verdaccio-starter/src/index.ts

```diff
@@ -129,12 +133,44 @@ function computeVerdaccioConfig(packagesToProxy: Set<string>) {
 }
 
 function doPublishProxiedPackages(
-  computedPackagesToProxy: Set<string>,
+  computedPackagesToProxy: Map<string, string[]>,
   verdaccioUrl: string,
   npmConfigPath: string
 ) {
```

Comment:
I feel like this would be easier to understand if it took an ordered list like before, but a separate function (e.g., `topologicalSort()`) was responsible for generating a properly ordered array.

Reply:
Probably, but it's more complex to do, and I don't think it's a good idea to spend time doing it, at least now